1_wqpdata/src/wqp_pull.R

```diff
@@ -16,10 +16,11 @@ filter_partitions <- function(partitions, pull_task) {
 # so that we know whether to re-pull a partition based solely on whether the
 # contents of that row have changed. each row should therefore include the lists
 # of sites and characteristic names.
-partition_inventory <- function(inventory_ind, wqp_pull, wqp_state_codes, wqp_codes) {
+partition_inventory <- function(ind_file, inventory_ind, wqp_pull, wqp_state_codes, wqp_codes) {
   # read in the inventory, which includes all dates and is therefore a superset
   # of what we'll be pulling for a specific date range
-  scipiper::sc_retrieve(inventory_ind)
+
+  #scipiper::sc_retrieve(inventory_ind)
```

Comment:
This line is necessary to ensure that `scipiper::as_data_file(inventory_ind)` exists on whatever computer is running `partition_inventory`. It'll work on your computer without this line but wouldn't work on others.

A potentially cleaner pattern is to use the output of sc_retrieve to get the data file name rather than calling as_data_file separately:
```r
inventory <- scipiper::sc_retrieve(inventory_ind) %>%
  feather::read_feather() %>%
  select(Constituent, Site=MonitoringLocationIdentifier, StateCode, resultCount)
```